package.json

```diff
@@ -62,6 +62,11 @@
     "knex": "0.13.0",
     "knex-migrator": "2.0.16",
     "lodash": "4.17.4",
+    "markdown-it": "8.3.1",
+    "markdown-it-footnote": "3.0.1",
+    "markdown-it-lazy-headers": "^0.1.3",
```

Comment:
We usually pin versions strict on the server. I wonder...now that we have a yarn lock file and e.g. `markdown-it-lazy-headers 0.1.4` get's released, nobody would auto install this version until i update the dependencies in our yarn.lock. So maybe we can make version pinning a bit loose again (on `LTS` we hadn't used shrinkwrap and that's why it was imported to pin strict)